includes/class-amp-story-post-type.php

```diff
@@ -305,4 +397,54 @@ public static function admin_body_class( $class ) {
 		$class .= ' edit-story ';
 		return $class;
 	}
+
+	/**
+	 * Filter the allowed tags for KSES to allow for amp-story children.
+	 *
+	 * @param array $allowed_tags Allowed tags.
+	 * @return array Allowed tags.
+	 */
+	public static function filter_kses_allowed_html( $allowed_tags ) {
```

Comment:
Like looks like repeated code. Instead of repeating code, we should use a [Abstraction Class](https://www.php.net/manual/en/language.oop5.abstract.php) or [traits](https://www.php.net/manual/en/language.oop5.traits.php) to stop repeated methods.

Reply:
Actually both the story components and allowed tags are different. It's not final yet either and there's a possibility that we can remove this from the legacy post if editing (thus saving) will be disabled anyway. Will keep it like this for now until things are more clear.